client/two-column-tasks/extended-task.tsx

```diff
@@ -88,7 +88,7 @@ const ExtendedTask: React.FC< TasksProps > = ( { query } ) => {
 		return <TasksPlaceholder query={ query } />;
 	}
 
-	if ( currentTask ) {
+	if ( currentTask && rednerTask ) {
```

Comment:
I'm not too familiar with the new task list structure, but at a quick glance this workaround seems like it may be covering up another issue we have with `currentTask`.

I think the solution in this PR is a good short-term fix though and I don't think it will expose anything too bad since task IDs should not be identical in either list, but something we might need to keep in mind going forward.

Reply:
Totally agree. This is definitely a short-term fix as we're trying to include the two-cols experiment in 3.1.0.  I'll look into a better solution if we decide to go with the new two-col tasklist design. Thank you for the suggestion 👍